recipes/id3v2lib/all/conanfile.py

```diff
@@ -0,0 +1,79 @@
+from conan import ConanFile
+from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout
+from conan.tools.files import copy, get
+from conan.errors import ConanInvalidConfiguration
+import os
+
+required_conan_version = ">=1.53.0"
+
+
+class LibwebmConan(ConanFile):
+    name = "id3v2lib"
+    description = "id3v2lib is a library written in C to read and edit id3 tags from mp3 files."
+    topics = ("conan", "id3", "tags", "mp3", "container", "media", "audio")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/larsbs/id3v2lib"
+    license = "BSD-2-Clause"
+
+    settings = "os", "arch", "compiler", "build_type"
+
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+    }
+
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+    }
+
+    def validate(self):
+        # An issue has been opened to discuss supporting MSVC:
+        # https://github.com/larsbs/id3v2lib/issues/48
+        if self.settings.compiler == "msvc" or str(self.settings.compiler) == "Visual Studio":
```

Comment:
```suggestion
        if is_msvc(self):
```

Reply:
Oh wow, how did I miss that? That's **much** better, thanks 👍🏻